web/templates/compose_banner/upload_banner.hbs

```diff
@@ -1,5 +1,8 @@
 <div class="upload_banner file_{{file_id}} compose_banner {{banner_type}}">
     <div class="moving_bar"></div>
     <p class="upload_msg">{{banner_text}}</p>
-    <a role="button" class="zulip-icon zulip-icon-close compose_banner_close_button"></a>
+    <div class="upload_banner_actions_button">
+        <button class="upload_banner_cancel_button">{{t 'Cancel' }}</button>
+        <a role="button" class="zulip-icon zulip-icon-close compose_banner_close_button"></a>
+    </div>
```

Comment:
This looks wrong, please take a look at other compose banners.

Reply:
Fixed this.